src/utils/schema.ts

```diff
@@ -0,0 +1,234 @@
+import { BigNumber } from 'bignumber.js'
```

Comment:
Which files had their content moved here?

Reply:
this file is mostly new (took methods we wrote for atomicizer logic out of wyvern-schemas), aside from `encodeAtomicizedTransfer` and `encodeTransferCall`, which were moved out of utils.ts